modules/globebrowsing/src/tileprovider/tileproviderbylevel.cpp

```diff
@@ -27,11 +27,28 @@
 #include <openspace/documentation/documentation.h>
 
 namespace {
+    // This tile provider will switch between different tile providers specified within
+    // based on the level of detail that is requested by the Globe. All other things being
+    // equal, this corresponds to the distance of the camera to the planet, with a closer
+    // distance resulting in a higher lever. Due to technical reasons, the available
+    // levels are in the range [2, 22] and each increase in levels corresponds to a
+    // doubling in the effective resolution. For a given requested level, the tile
+    // provider that has the largest `MaxLevel` that is not greater than the requested
+    // level will be used.
```

Comment:
I found this text a bit hard to understand / technical. I tried to simplify this text a bit... a suggestion (is it even correct?)

This tile provider will switch between different tile providers based on the distance of the camera to the globe. Each tile provider will specify the distances in which they are valid. The distance is specified in _levels_. A smaller distance corresponds to a larger level number. The levels are in the range [2, 22]. Each increase in level corresponds to a doubling in the texture resolution. For a given requested level, the tile provider that has the largest `MaxLevel` that is not greater than the requested level will be used.

Reply:
The reason its a bit strange is that it doesn't only depend on the distance of the camera as the "TargetLevelOfDetail" slider has a direct impact as well.  And it's on the list to make a tile provider that actually works on camera distance.

How about:

This tile provider will switch between different tile providers specified within based on the required level of detail. The level of details mostly depends on the distance of the camera to the planet, with a closer distance resulting in a higher level. The levels are in the range [2, 22]. Each increase in level corresponds to a doubling in the texture resolution. For a given requested level, the tile provider that has the largest MaxLevel that is not greater than the requested level will be used.
```